examples/thoughts/ext/Tag.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import ColorHash from 'color-hash'
+import { Link } from 'react-router-dom'
+
+import './Tag.css'
+
+const getClasses = (href, isActive, isDeletable) => `${href ? 'link' : ''} ${isActive ? 'active' : ''} ${isDeletable ? 'deletable' : ''}`
```

Comment:
Let's break this into two lines. Actually, I would probably do it with array + join and then put every class into its own line. I guess you didn't optimize this too much since you respect this to be removed anyway once styled components arrive.

Reply:
Yep, I like the array join approach. Definitely something that will look a lot cleaner once we start bringing styled-components in though.